src/main/java/org/testcontainers/jooq/codegen/Plugin.java

```diff
@@ -80,9 +84,9 @@ public void execute() throws MojoExecutionException {
                     || this.jdbc.getUrl() == null
                     || this.jdbc.getUsername() == null
                     || this.jdbc.getPassword() == null) {
-                container = DatabaseProvider.getDatabaseContainer(database);
+                var container = DatabaseProvider.getDatabaseContainer(database);
                 container.start();
-
+                oContainer = Optional.ofNullable(container);
```

Comment:
this optional looks redundant, because if container could be null then we will have NPE in line 88 where 
`container.start()` is being executed.

Reply:
Hi @bitxon  , actually this method cannot return null by contract.
But i added extra check that required parameter 'database.type' has been specified. 
thank you!

Reply:
If `container` could not be null, then Optional should not be used. (Just recommendation)

Reply:
i wanted to get rid of nullable container, but did it not the perfect way. Agreed, reverted changes.Will refactor it separately.
Thanks @bitxon